torch/_torch_docs.py

```diff
@@ -8928,6 +8928,50 @@ def merge_dicts(*dicts):
 """.format(**factory_common_args))
 
 
+add_docstr(torch.kaiser_window,
+           """
+kaiser_window(window_length, beta, periodic=True, dtype=None, \
+layout=torch.strided, device=None, requires_grad=False) -> Tensor
+""" + r"""
+Kaiser window function.
```

Comment:
Would a more accurate description be something like, "Computes symmetric samples from the Kaiser windowing function"?

Reply:
I don't think I agree with this, since you can compute non symmetric samples with this function (in fact the default is to do that).

Reply:
That makes sense, I realize now what I suggested is a problem. What I'm trying to get at is something more like, "samples symmetrically from..." as in this [Wikipedia article](https://en.wikipedia.org/wiki/Kaiser_window#:~:text=It%20is%20a%20one%2Dparameter,which%20is%20difficult%20to%20compute.). But all the other documentation, like NumPy and matlab, seem to just refer to this as computing the Kaiser window function, which doesn't seem correct but is probably good enough for now.

What about just, "Computes the Kaiser window with window length :attr:\`window_length\` and shape parameter :attr:\`beta\`."? It's nice to include how the positional arguments relate to the function in the opening sentence. I don't love the user of the term "shape factor" for beta, but it seems reasonable and consistent with other descriptions.